### DISCUSSION TITLE
GitHub repository

### DISCUSSION THREAD

**Discussion by MissMeJully** [community/community#52812] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi there, I'm new to GitHub and I'm not sure how to set up a repository. Can someone guide me through the process?

**Comment by opegamanalitarok** [comment, marked as answer]
Hello, my friend's friend, 

Of course, happy to help! Here are the steps to create a new repository on GitHub:

1. Log in to your GitHub account.
2. Click on the 'Repositories' icon in the top right corner and select 'New'.
3. Choose a name for your repository and add a description (optional).
4. Select the type of repository you want to create - public or private.
5. Add a README file (optional).
6. Choose a license (optional).
7. Click on the 'Create repository' button.

And that's it! Your new repository is now set up and ready to go. Let me know if you have any further questions or if there's anything else I can help with.

**Reply by kelvingithub2023** [reply]
I have done creating a repository but the Link is not working with Notion.
It may need to be published?
How to activate the repository created?
Thank you.
